### ISSUE TITLE
twisted.python.test.module_helpers import twisted.python.modules

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6041]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6041|
|Type|enhancement|
|Created|2012-10-03 12:10:21Z|
|Branch|https://github.com/twisted/twisted/tree/modulehelpers-py3-6041|

Due to porting dependencies (`twisted.python.modules` hasn't yet been ported to Python 3), it would be useful for `twisted.python.test.module_helpers` to not import `modules`. The one function that depends on it is in any case only used in `twisted.test.test_modules` and can be easily moved.

<details><summary>Searchable metadata</summary>

```
trac-id__6041 6041
type__enhancement enhancement
reporter__itamar itamar
priority__normal normal
milestone__Python_3_3_Minimal Python 3.3 Minimal
branch__branches_modulehelpers_py3_6041 branches/modulehelpers-py3-6041
branch_author__itamarst itamarst
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1349266221000000 1349266221000000
changetime__1349278601000000 1349278601000000
version__None None
owner__itamar itamar

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_1"></a>|@itamarst commented|
|-|-|

(In [35896]) Branching to 'modulehelpers-py3-6041'

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set owner to @exarkun|
|-|-|

http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/modulehelpers-py3-6041 has started.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @itamarst|
|-|-|

Thanks.  Looks good.  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_4"></a>|@itamarst set status to `closed`|
|-|-|

(In [35902]) Merge modulehelpers-py3-6041.

Author: itamar
Review: exarkun
Fixes: [#6041](https://github.com/twisted/twisted/issues/6041)


twisted.python.test.modules_helpers now works on Python 3.
